### PR TITLE
*: add docs home and tidb docs learning path

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Add docs home and TiDB home (TiDB docs learning path).

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/9560
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: _docHome.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+<DocHomeSection label="TiDB Cloud" anchor="tidb-cloud" id="tidb-cloud">
+
+TiDB Cloud 是全托管的数据库即服务 (Database-as-a-Service, DBaaS) 产品，依托于公有云提供开箱即用的 TiDB 服务。TiDB Cloud 屏蔽 TiDB 数据库部署、运维和性能调优的复杂性，您可以通过界面上几下点击就可以快速创建和管理 TiDB 实例，让您专注于自身业务应用程序的开发，降低总体拥有成本。适用于中国出海企业和开发者。
+
+<DocHomeCardContainer>
+
+<DocHomeCard href="https://docs.pingcap.com/tidbcloud" icon="doc2" label="TiDB Cloud 文档">
+
+查看 TiDB Cloud 的全部产品文档（英文版）
```
Comment (lilin90) [inline _docHome.md:50]:
```suggestion
查看 TiDB Cloud 产品的全部文档（英文版）
```

File: _docHome.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart" icon="cloud5" label="TiDB Cloud 快速上手">
+
+快速体验和上手 TiDB Cloud
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/tidbcloud/tidb-cloud-poc" icon="cloud3" label="TiDB Cloud 概念验证（PoC）">
```
Comment (lilin90) [inline _docHome.md:60]:
```suggestion
<DocHomeCard href="https://docs.pingcap.com/tidbcloud/tidb-cloud-poc" icon="cloud3" label="TiDB Cloud 概念验证 (PoC)">
```

File: _docHome.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+帮助您快速完成 TiDB Cloud 概念验证 (Proof of Concept, PoC)
+
+</DocHomeCard>
+
+</DocHomeCardContainer>
+
+体验云原生、分布式的实时 HTAP 数据库 TiDB Cloud
```
Comment (lilin90) [inline _docHome.md:68]: 加了一个词，以便与 TiDB 区分
```suggestion
体验全托管的云原生、分布式、实时 HTAP 数据库 TiDB Cloud
```

File: _docHome.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+<DocHomeCard href="https://asktug.com" icon="asktug" label="Ask TiDB User Group">
+
+互助交流，有问有答
+
+</DocHomeCard>
+
+<DocHomeCard href="https://ossinsight.io" icon="cloud6" label="OSS Insight">
+
+一款强大有趣的洞察工具，帮您深入分析 GitHub 库
```
Comment (lilin90) [inline _docHome.md:130]:
```suggestion
一款强大有趣的洞察工具，帮您深入分析 GitHub 仓库
```

File: _index.md [hunk 5]
```diff
@@ -1,117 +1,124 @@
...
-summary: 了解 TiDB 数据库。
-aliases: ['/docs-cn/dev/','/docs-cn/','/docs-cn/stable/']
+title: TiDB 产品文档
+aliases: ["/docs/dev/", "/docs/dev/adopters/", "/tidb/dev/adopters"]
+hide_sidebar: true
+hide_commit: true
 ---
 
-<!-- markdownlint-disable MD046 -->
+<LearningPathContainer platform="tidb" title="TiDB" subTitle="TiDB 是 PingCAP 公司自主设计、研发的开源分布式关系型数据库。您可以在这里查看操作指南、示例、参考等产品文档。">
```
Comment (lilin90) [inline _index.md:8]: 不确定增加的字数会不会影响正常显示
```suggestion
<LearningPathContainer platform="tidb" title="TiDB" subTitle="TiDB 是 PingCAP 公司自主设计、研发的开源分布式关系型数据库。您可以在这里查看概念介绍、操作指南、应用开发、参考等产品文档。">
```

File: _index.md [hunk 6]
```diff
@@ -1,117 +1,124 @@
...
+
+<LearningPath label="调优" icon="tidb-cloud-tune">
+
+[调优概述](https://docs.pingcap.com/zh/tidb/stable/performance-tuning-overview)
+
+[调优方法](https://docs.pingcap.com/zh/tidb/stable/performance-tuning-methods)
+
+[调优实践](https://docs.pingcap.com/zh/tidb/stable/performance-tuning-practices)
+
+[调优操作系统性能参数](https://docs.pingcap.com/zh/tidb/stable/tune-operating-system)
```
Comment (lilin90) [inline _index.md:94]: 后三项调优显示在同一行，保持用词顺序统一
```suggestion
[操作系统性能参数调优](https://docs.pingcap.com/zh/tidb/stable/tune-operating-system)
```

File: _index.md [hunk 7]
```diff
@@ -1,117 +1,124 @@
...
-<ColumnTitle>监控告警</ColumnTitle>
+[环境与系统配置检查](https://docs.pingcap.com/zh/tidb/stable/check-before-deployment)
 
-- [监控框架概述](/tidb-monitoring-framework.md)
-- [监控 API](/tidb-monitoring-api.md)
-- [部署监控](/deploy-monitoring-services.md)
-- [将 Grafana 监控数据导出成快照](/exporting-grafana-snapshots.md)
-- [TiDB 集群报警规则与处理方法](/alert-rules.md)
-- [TiFlash 报警规则与处理方法](/tiflash/tiflash-alert-rules.md)
+[部署 TiDB 集群（推荐方式）](https://docs.pingcap.com/zh/tidb/stable/production-deployment-using-tiup)
```
Comment (lilin90) [inline _index.md:46]: 部署部分也加一下在 Kubernetes 上部署？

File: _index.md [hunk 8]
```diff
@@ -1,117 +1,124 @@
...
+
+[Dumpling](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview)
+
+[TiDB Lightning](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview)
+
+[Data Migration](https://docs.pingcap.com/zh/tidb/stable/dm-overview)
+
+[Backup & Restore (BR)](https://docs.pingcap.com/zh/tidb/stable/backup-and-restore-overview)
+
+[TiDB Binlog](https://docs.pingcap.com/zh/tidb/stable/tidb-binlog-overview)
```
Comment (lilin90) [inline _index.md:114]: 现在建议使用 TiCDC 替代 TiDB Binlog，这里可以不单独列出来

File: _index.md [hunk 9]
```diff
@@ -1,117 +1,124 @@
...
+
+[TiCDC](https://docs.pingcap.com/zh/tidb/stable/ticdc-overview)
+
+[PingCAP Clinic](https://docs.pingcap.com/zh/tidb/stable/clinic-introduction)
+
+[sync-diff-inspector](https://docs.pingcap.com/zh/tidb/stable/sync-diff-inspector-overview)
```
Comment (lilin90) [inline _index.md:120]: 这个可以考虑换成 TiUniManager？

File: _index.md [hunk 10]
```diff
@@ -1,117 +1,124 @@
...
+
+</LearningPath>
+
```
Comment (lilin90) [inline _index.md:123]: 另外，看下用户常访问的 TiFlash 能否也提供一个入口

File: _index.md [hunk 11]
```diff
@@ -1,117 +1,124 @@
...
-<ColumnTitle>监控告警</ColumnTitle>
+[环境与系统配置检查](https://docs.pingcap.com/zh/tidb/stable/check-before-deployment)
 
-- [监控框架概述](/tidb-monitoring-framework.md)
-- [监控 API](/tidb-monitoring-api.md)
-- [部署监控](/deploy-monitoring-services.md)
-- [将 Grafana 监控数据导出成快照](/exporting-grafana-snapshots.md)
-- [TiDB 集群报警规则与处理方法](/alert-rules.md)
-- [TiFlash 报警规则与处理方法](/tiflash/tiflash-alert-rules.md)
+[部署 TiDB 集群（推荐方式）](https://docs.pingcap.com/zh/tidb/stable/production-deployment-using-tiup)
```
Comment (hfxsd) [inline _index.md:46]:
```suggestion
[使用 TiUP 部署 TiDB 集群（推荐方式）](https://docs.pingcap.com/zh/tidb/stable/production-deployment-using-tiup)

[在 Kubernetes 上部署 TiDB](https://docs.pingcap.com/zh/tidb/stable/tidb-in-kubernetes)

```

File: _index.md [hunk 12]
```diff
@@ -1,117 +1,124 @@
...
+
+[Dumpling](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview)
+
+[TiDB Lightning](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview)
+
+[Data Migration](https://docs.pingcap.com/zh/tidb/stable/dm-overview)
+
+[Backup & Restore (BR)](https://docs.pingcap.com/zh/tidb/stable/backup-and-restore-overview)
+
+[TiDB Binlog](https://docs.pingcap.com/zh/tidb/stable/tidb-binlog-overview)
```
Comment (hfxsd) [inline _index.md:114]:
```suggestion
```

File: _index.md [hunk 13]
```diff
@@ -1,117 +1,124 @@
...
+
+[TiCDC](https://docs.pingcap.com/zh/tidb/stable/ticdc-overview)
+
+[PingCAP Clinic](https://docs.pingcap.com/zh/tidb/stable/clinic-introduction)
+
+[sync-diff-inspector](https://docs.pingcap.com/zh/tidb/stable/sync-diff-inspector-overview)
```
Comment (hfxsd) [inline _index.md:120]:
```suggestion
[TiUniManager](https://docs.pingcap.com/zh/tidb/stable/tiunimanager-overview)
```

File: _index.md (hunk 10 above)
Comment (hfxsd) [reply _index.md:123]: 了解 下面的 第三个

File: _index.md [hunk 14]
```diff
@@ -1,117 +1,124 @@
...
+
```
Comment (hfxsd) [inline _index.md:121]:
```suggestion

[TiDB Operator](https://docs.pingcap.com/zh/tidb/stable/tidb-operator-overview)

[TiSpark](https://docs.pingcap.com/zh/tidb/stable/tispark-overview)

```

File: _index.md [hunk 15]
```diff
@@ -1,117 +1,124 @@
...
-- [TiUP 常用运维操作](/maintain-tidb-using-tiup.md)
-- [TiFlash 常用运维操作](/tiflash/maintain-tiflash.md)
+<LearningPath label="部署" icon="deploy">
 
-</NavColumn>
+[软硬件环境需求](https://docs.pingcap.com/zh/tidb/stable/hardware-and-software-requirements)
 
-<NavColumn>
-<ColumnTitle>监控告警</ColumnTitle>
+[环境与系统配置检查](https://docs.pingcap.com/zh/tidb/stable/check-before-deployment)
```
Comment (hfxsd) [inline _index.md:44]:
```suggestion
```

File: _index.md [hunk 16]
```diff
@@ -1,117 +1,124 @@
...
```
Comment (hfxsd) [inline _index.md:45]:
```suggestion
```

File: _index.md (hunk 9 above)
Comment (TomShawn) [reply _index.md:119]: +1

File: _docHome.md [hunk 17]
```diff
@@ -0,0 +1,138 @@
+---
+title: PingCAP 文档中心
+hide_sidebar: true
+hide_commit: true
+hide_leftNav: true
+---
+
+<DocHomeContainer title="PingCAP 文档中心" subTitle="欢迎来到 PingCAP 文档中心！我们为您使用 TiDB 产品、迁移数据、基于数据库开发应用等场景提供了丰富的操作指南、详实的参考资料，帮您顺利完成操作，为您排忧解难。">
```
Comment (shichun-0415) [inline _docHome.md:8]:
```suggestion
<DocHomeContainer title="PingCAP 文档中心" subTitle="欢迎来到 PingCAP 文档中心！我们为您提供了丰富的操作指南和详实的参考资料，助您轻松上手 TiDB 产品，顺利完成数据迁移和基于数据库的应用开发等操作。">
```

File: _docHome.md [hunk 18]
```diff
@@ -0,0 +1,138 @@
...
+
+<DocHomeSection label="TiDB" anchor="tidb" id="tidb">
+
+TiDB 是 PingCAP 公司自主设计、研发的开源分布式关系型数据库，是一款同时支持在线事务处理与在线分析处理 (Hybrid Transactional and Analytical Processing, HTAP) 的融合型分布式数据库产品，具备水平扩容或者缩容、金融级高可用、实时 HTAP、云原生的分布式数据库、兼容 MySQL 5.7 协议和 MySQL 生态等重要特性，支持在本地和云上部署。
+
+<DocHomeCardContainer>
+
+<DocHomeCard href="/tidb/stable" icon="doc1" label="TiDB 文档">
+
+查看 TiDB 产品的全部文档
```
Comment (shichun-0415) [inline _docHome.md:18]:
```suggestion
查看 TiDB 产品文档
```

File: _docHome.md [hunk 19]
```diff
@@ -0,0 +1,138 @@
...
+
+<DocHomeSection label="TiDB Cloud" anchor="tidb-cloud" id="tidb-cloud">
+
+TiDB Cloud 是全托管的数据库即服务 (Database-as-a-Service, DBaaS) 产品，依托于公有云提供开箱即用的 TiDB 服务。TiDB Cloud 让数据库部署、运维和性能调优变得轻松简单，通过界面上的几次点击即可快速创建和管理 TiDB 集群，让您可以专注于自身业务。适用于中国出海企业和开发者。
+
+<DocHomeCardContainer>
+
+<DocHomeCard href="https://docs.pingcap.com/tidbcloud" icon="doc2" label="TiDB Cloud 文档">
+
+查看 TiDB Cloud 产品的全部文档（英文版）
```
Comment (shichun-0415) [inline _docHome.md:50]: 反正别人打开能看到是英文版，我们也不用强调是什么语言吧，而且这么强调别人也不明白我们只有英文版，没有中文版。
```suggestion
查看 TiDB Cloud 产品文档
```

File: _docHome.md [hunk 20]
```diff
@@ -0,0 +1,138 @@
...
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart" icon="cloud5" label="TiDB Cloud 快速上手">
+
+快速体验和上手 TiDB Cloud
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/tidbcloud/tidb-cloud-poc" icon="cloud3" label="TiDB Cloud 概念验证 (PoC)">
```
Comment (shichun-0415) [inline _docHome.md:60]: 下面的副标题有 PoC 的解释，大标题就不用放了。
```suggestion
<DocHomeCard href="https://docs.pingcap.com/tidbcloud/tidb-cloud-poc" icon="cloud3" label="TiDB Cloud PoC">
```

File: _docHome.md [hunk 21]
```diff
@@ -0,0 +1,138 @@
...
+
+<DocHomeCard href="https://pingcap.com/zh/education" icon="cloud1" label="PingCAP Education">
+
+提供众多免费课程，助您深入学习 TiDB，成为 TiDB 技术专家
+
+</DocHomeCard>
+
+<DocHomeCard href="https://tidb.net" icon="doc9" label="TiDB 社区">
+
+开发者、用户、Contributor、合作伙伴一起建立的分享、学习平台
```
Comment (shichun-0415) [inline _docHome.md:112]: 按逻辑，一般是学习在前，分享在后
```suggestion
开发者、用户、Contributor、合作伙伴一起建立的学习和分享平台
```

File: _docHome.md [hunk 22]
```diff
@@ -0,0 +1,138 @@
...
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/zh/tidb/stable/dev-guide-sample-application-java" icon="java" label="Java">
+
+使用 TiDB 和 Java 来构造一个简单的 CRUD 应用程序
+
+</DocHomeCard>
+
+<DocHomeCard href="https://docs.pingcap.com/zh/tidb/stable/dev-guide-outdated-for-go-sql-driver-mysql" icon="go" label="Golang">
```
Comment (qiancai) [inline _docHome.md:90]:
```suggestion
<DocHomeCard href="https://docs.pingcap.com/zh/tidb/stable/dev-guide-sample-application-golang" icon="go" label="Golang">
```

File: _docHome.md [hunk 23]
```diff
@@ -0,0 +1,138 @@
...
+
+在生产环境中部署本地 TiDB 集群
+
+</DocHomeCard>
+
+</DocHomeCardContainer>
+
+体验全新的一栈式实时 HTAP 数据库
+
+<a href="https://pingcap.com/zh/product-community" class="button" target="_blank" referrerpolicy="no-referrer-when-downgrade">下载</a>
```
Comment (qiancai) [inline _docHome.md:38]:
```suggestion
<a href="https://pingcap.com/zh/product-community" class="button" target="_blank" referrerpolicy="no-referrer-when-downgrade">免费下载</a>
```

File: _docHome.md (hunk 19 above)
Comment (hfxsd) [reply _docHome.md:50]: 先保留英文版，以免用户点开后发现是英文有疑问。

File: _docHome.md (hunk 18 above)
Comment (hfxsd) [reply _docHome.md:18]: 先保留现状

**Review by TomShawn — APPROVED** [review]
🎉

File: _docHome.md [hunk 24]
```diff
@@ -0,0 +1,138 @@
...
+
+<DocHomeCard href="https://asktug.com" icon="asktug" label="Ask TiDB User Group">
+
+互助交流，有问有答
+
+</DocHomeCard>
+
+<DocHomeCard href="https://ossinsight.io" icon="cloud6" label="OSS Insight">
+
+一款强大有趣的洞察工具，帮您深入分析 GitHub 仓库
```
Comment (lilin90) [inline _docHome.md:130]:
```suggestion
一款由 TiDB Cloud 驱动的强大有趣的洞察工具，帮您深入分析 GitHub 仓库
```

**Review by lilin90 — COMMENTED** [review]
1. This PR will be merged to the master branch, while all links are "stable" ones. The versions are not consistent.
2. If the relative path works, please use it instead for all related links. When changes are cherry-picked to other branches, you don't need to update links.
----

Since the "stable" links are what are displayed by default when users access the docs, we'll use "stable" links in the files, but will update the version info in links for other TiDB versions.

File: _index.md [hunk 25]
```diff
@@ -1,117 +1,128 @@
...
+[TiUP](https://docs.pingcap.com/zh/tidb/stable/tiup-overview)
+
+[Dumpling](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview)
+
+[TiDB Lightning](https://docs.pingcap.com/zh/tidb/stable/tidb-lightning-overview)
+
+[Data Migration](https://docs.pingcap.com/zh/tidb/stable/dm-overview)
+
+[Backup & Restore (BR)](https://docs.pingcap.com/zh/tidb/stable/backup-and-restore-overview)
+
```
Comment (hfxsd) [inline _index.md:114]:
```suggestion
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**